sip/execution_framework_interface/docker_api/client/test/test_docker_client.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+"""Test for docker client API"""
+
+import logging
+import time
+
+from ..docker_client import DockerClient
+
+logging.basicConfig(level='DEBUG')
+
+DC = DockerClient()
+
+
+def test_create_services():
+    """ Test function for create services"""
+
+    # Remove all services and volumes
+    DC.delete_all_services()
+    assert not DC.get_service_list()
+    time.sleep(2)
+    DC.delete_all_volumes()
+    assert not DC.get_volume_list()
+
+    # Passing in a test compose file
+    filename = ["compose-file/docker-compose.visibility.yml"]
+    DC.create_services(filename)
+    assert len(DC.get_service_list()) == 1
```

Comment:
While it is useful to test that the service exists, it might be even better to test that the service is running as well.